extensions/config/portability-config-yaml/src/main/java/org/dataportabilityproject/config/yaml/YamlSettingsExtension.java

```diff
@@ -63,11 +69,13 @@ public void initialize(ExtensionContext context) {
         .add(EXTENSION_SETTINGS_PATH)
         .build();
     InputStream in = ConfigUtils.getCombinedInputStream(settingsFiles);
-    parse(in);
+    parseSimple(in);
+    in = ConfigUtils.getCombinedInputStream(ImmutableList.of(RETRY_LIBRARY_PATH));
```

Comment:
I was a little confused by re-using the same in object. Maybe these read input stream lines can be broken up into different method?